lib/shared/types/src/types/apis/sdk/clientSDKAPI.ts

```diff
@@ -38,6 +39,7 @@ export class DVCAPIUser {
      * Must be defined if `isAnonymous = false`
```

Comment:
This comment implies that this field should be optional, for the isAnonymous: true scenario?